src/app/(routes)/shopping-mall/context/category-context.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactNode, createContext, useMemo } from "react";
+import { Item } from "../types";
+import data from "../data.json";
+import { Brand, getBrandCategories } from "./get-brand-categories";
+import { Country, getCountryCategories } from "./get-country-categories";
+import { Word, getWordCategories } from "./get-word-categories";
+
+export type Categories = Record<Country, Item[]> &
+  Record<Word, Item[]> &
+  Record<Brand, Item[]>;
+
+export const initialCategories: Categories = {
+  czech: [],
+  polish: [],
+  parrot: [],
+  sheep: [],
+  hourglass: [],
+  skincare: [],
+  rituals: [],
+  parfume: [],
+  slytherin: [],
+  fox: [],
+  sunflower: [],
+  puzzle: [],
+};
+
+// @ts-ignore
```

Comment:
Much code smell, very not safe... :see_no_evil:

Reply:
Would you know how to avoid it? :smirk:

Reply:
Took me longer than I would like to admit!

`const shoppingMallItems = (data as { items: Item[] }).items;`

I could also just `require("file.json")` but I've read that that is a bad practise?